lib/ae_mdw_web/router.ex

```diff
@@ -94,6 +94,7 @@ defmodule AeMdwWeb.Router do
       get "/stats/total", StatsController, :total_stats
       get "/stats/delta", StatsController, :delta_stats
       get "/stats/miners", StatsController, :miners_stats
+      get "/stats/contracts-count", StatsController, :contracts_count_stats
```

Comment:
Can we rename this route to just `/status/contracts`, none of the other routes include the word count.